__docs__/wonder-blocks-tabs/navigation-tab-item.stories.tsx

```diff
@@ -41,3 +45,46 @@ export const Default: StoryComponentType = {
         },
     },
 };
+
+/**
+ * When a `Link` component is passed in for the `children` prop,
+ * `NavigationTabItem` will inject props for the `Link`.
+ *
+ * For specific use cases where the `Link` component is wrapped by another
+ * component (like a `Tooltip` or `Popover`), a render function can be used
+ * instead. The render function provides the Link props that should be applied
+ * to the Link component. The Link props contains styles and attributes for
+ * accessibility like `aria-current`.
+ */
+export const ChildrenRenderFunction = () => {
```

Comment:
Note: This example shows that the NavigationTabItem supports flexible `children`. 

I created WB-1901 to investigate a11y considerations using NavigationTabs and Tooltip/Popover together.